assets/css/style.css

```diff
@@ -456,3 +453,596 @@ footer a {
+  }
+
+  .surport-text .items {
+    width: 446px;
+    height: auto;
+    display: flex;
+    padding: 0;
+    margin: 24px 0;
+    flex-grow: 0;
+    flex-wrap: wrap;
+    gap: 10px;
+  }
+
+  .item {
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;
+    width: 90px;
+    margin: 0 24px;
+    padding: 5px;
+  }
+
+  .list-items {
+    list-style: none;
+  }
+
+  .item h3 {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 29px;
+    padding-bottom: 10px;
+    align-items: center;
+    letter-spacing: 0.03em;
+    color: #9490ff;
+  }
+
+  .surport-text .social-items {
+    display: flex;
+    width: 1023px;
+    margin: 88px 0;
+  }
+
+  footer .social {
+    display: flex;
+    gap: 15px;
+    margin-top: 5px;
+    padding: 0;
+    width: 330px;
+    justify-content: flex-end;
+  }
+
+  .surport-text .social {
+    display: flex;
+  }
+
+  .surport-text .social img:hover {
+    background: #9490ff;
+  }
+
+  .surport-text .more {
+    display: flex;
+    margin: 0 auto;
+    margin-top: 25px;
+    padding: 10px;
+    gap: 5px;
+  }
+
+  .contact-sec p {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    word-wrap: break-word;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #f5f5ff;
+    margin: 24px 0;
+  }
+
+  .surport-text .more p {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #a2a3ae;
+    margin: 0 12px;
+  }
+
+  /* Works section */
+  .work {
+    width: 1156px;
+    display: flex;
+    flex-direction: column;
+    flex-wrap: wrap;
+    align-items: center;
+    padding: 114px 0;
+    margin: 88px 0;
+    position: relative;
+    left: 142px;
+  }
+
+  .workg {
+    background-image: url(../images/Ellipse19.svg), url(../images/Ellipse20.svg), url(../images/Ellipse21.svg), url(../images/Ellipse22.svg);
+    background-repeat: no-repeat, no-repeat, no-repeat, no-repeat;
+    background-position-y: 15%, 47%, 66%, 81%;
+    background-position-x: -10%, 100%, 0%, 100%;
+  }
+
+  .worktitle {
+    width: 1156px;
+    height: 155px;
+    margin: 88px 0;
+  }
+
+  .worktitle h1 {
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 64px;
+    text-align: left;
+    position: relative;
+    left: 0;
+    color: #9390ff;
+    flex: none;
+    flex-grow: 0;
+    margin: 24px 0;
+    width: auto;
+    animation-name: slidein;
+    animation-duration: 3s;
+  }
+
+  @keyframes slidein {
+    from {
+      margin-left: 100%;
+      width: 300%;
+    }
+
+    to {
+      margin-left: 0%;
+      width: 100%;
+    }
+  }
+
+  .worktitle h1 br {
+    display: none;
+  }
+
+  .work .divider {
+    border: 1px solid #646369;
+  }
+
+  .work .cards {
+    display: grid;
+    grid-template-columns: 1fr;
```

Comment:
- [ ] Kindly decrease the space between cars following the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A10) . Keep coding 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/160214628-175fa57a-14bc-470f-a3d9-a03ff77b69fb.png)


Here you have a screenshot from the design:


![image](https://user-images.githubusercontent.com/74506933/160214636-a82f6e9c-88de-42ba-b879-91b25e4a0860.png)

Reply:
Updated changes as per review